content-src/experiments/color.yaml

```diff
@@ -0,0 +1,76 @@
+id: 17
+title: 'Color'
+slug: color
+is_featured: false
+platforms: ['addon', 'web']
+min_release: 60
+thumbnail: /static/images/experiments/color/color-thumbnail.png
+description: >
+  Show your colors. Create and share your own Firefox themes with just a few clicks.
+introduction: >
+  FIrefox Color makes it easy for anyone to change the look of Firefox.
+  It’s a simple tool that lets you choose different color combinations
+  and background textures for your own Firefox theme.
+image_twitter: /static/images/experiments/notes/notes-twitter.jpg
+image_facebook: /static/images/experiments/notes/notes-fb.jpg
+video_url: https://www.youtube.com/embed/vaFvr_70yxw
+changelog_url: 'https://github.com/mozilla/FirefoxColor/releases'
+contribute_url: 'https://github.com/mozilla/FirefoxColor'
+bug_report_url: 'https://github.com/mozilla/FirefoxColor/issues'
+discourse_url: 'https://discourse.mozilla-community.org/c/test-pilot/color'
+legal_notice: By proceeding, you agree to the <a>terms</a> and <a>privacy</a> policies of Test Pilot and the <a>Firefox Color privacy policy</a>.
+privacy_notice_url: 'https://github.com/mozilla/notes/blob/master/docs/metrics.md'
+measurements:
+  - >
+    Information about how you interact with the experiment, such as how you access
+    the feature, which buttons you click, and how frequently you use it.
+  - >
+    Information about whether you have the experimental add-on installed
+    while visiting the Firefox Color website.
+  - >
+    Information about which colors and backgrounds you use to create custom themes.
+    This helps us understand the most popular choices among experiment participants.
+xpi_url: 'https://testpilot.firefox.com/files/notes@mozilla.com/latest'
```

Comment:
I think this should be "FirefoxColor@mozilla.com/latest", per https://github.com/mozilla/FirefoxColor/blob/2852096f6e30627035c2ab852b74b719911d893b/package.json#L159